internal/handlers/jsonb1/msg_find.go

```diff
@@ -108,7 +108,6 @@ func (s *storage) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 			"ok", float64(1),
 		)},
 	})
-
```

Comment:
It should be possible to restart CI via UI: click "Checks" tab on PR, click "Re-run all jobs":
![image](https://user-images.githubusercontent.com/11512/157019666-12bfa1fa-045a-4f79-93b2-c13a3d045d43.png)

Reply:
thank you :)